src/main/java/com/commercetools/sync/products/utils/VariantReferenceResolutionUtils.java

```diff
@@ -252,5 +252,18 @@ private static boolean isValueAProductReference(@Nonnull final JsonNode valueAsJ
     return false;
   }
 
+  private static boolean isReferenceSetAttribute(@Nonnull final JsonNode attributeValueAsJsonNode) {
```

Comment:
Those 2 new methods are not used anywhere in this class.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/715/commits/cba77841bcade63cabcb1b8b573781410e10a758